LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) Ignacio Tripodi, Margaret Gruca
+Copyright (c) Edmund Miller
```

Comment:
not sure if authors should be removed, or just more added in this case (unless of course it is agreed with everyone) : https://nf-co.re/docs/contributing/guidelines/requirements/acknowledgements

Reply:
Yeah, this got changed because it kept breaking the template branch. Happy to add it back, I wasn't sure what to do about it. I think doing this has brought up some good points about what happens when a pipeline picks up new maintainers or the maintainers move on.